Sage/carbon
# docs(duelling-picklist): update demos to ensure they have valid HTML markup FE-5038

fix #4953

### Proposed behaviour


 Adds `ul` wrapper to the stories that only render the `PickListItem` components as they render as `li` elements which are invalid unless children of a list
 
### Current behaviour


`Add item`, `Remove item`, `Locked` and `Custom tooltip message` stories have invalid markup as render `li` element with no list parent

### Checklist




#### QA


### Additional context



### Testing instructions


Test `Add item`, `Remove item`, `Locked` and `Custom tooltip message` stories for any axe warnings

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
